### PR TITLE
Xenial removal and roadmap update

### PR DESCRIPTION
**Pull request by amanda11**

### CODE REVIEW

File: docs/source/roadmap.rst [hunk 1]
```diff
@@ -64,6 +58,13 @@ Submit a PR!
 Release History
 ---------------
 
+.. rubric:: Done in v3.5
+
+* **Ubuntu Focal:** Add support for Ubuntu 20.04, with Python 3.8
```
Comment (winem) [inline docs/source/roadmap.rst:63]: Do we want to mention mongo 4.4 here?

**Comment by winem** [comment]
https://github.com/StackStorm/st2docs/blob/master/docs/source/install/ansible.rst
https://github.com/StackStorm/st2docs/blob/master/docs/source/install/puppet.rst

Both of them mention U16 and the puppet link even U14. 

Do we want to remove this in this PR or just when the respective PRs in the ansible-st2 and puppet repository got merged? Beside that it else looks good to me!

**Review by winem — APPROVED** [review]

**Comment by amanda11** [comment]
I was going to go for updating puppet and ansible in docs - when we update the relevant installers.

File: docs/source/roadmap.rst (hunk 1 above)
Comment (amanda11) [reply docs/source/roadmap.rst]: Good point - updated

**Pull request merged**